frontend_tests/node_tests/recent_topics.js

```diff
@@ -260,7 +260,9 @@ function generate_topic_data(topic_info_array) {
     const selectors = [];
 
     for (const [stream_id, topic, unread_count, muted, participated] of topic_info_array) {
-        const topic_selector = $.create("#recent_topic:" + get_topic_key(stream_id, topic));
```

Comment:
This is all dead code, so I'll merge https://github.com/zulip/zulip/pull/17206 once the build passes.

Reply:
(merged, cc @amanagr)

Reply:
Rebased.